# Python - 3.4.3

Test.assert_equals(problem('hello'), 'Error')
Test.assert_equals(problem(1), 56)
